airbyte-cdk/python/.mypy.ini

```diff
@@ -1,4 +0,0 @@
-[mypy]
```

Comment:
replace the file with a top-level file. The top level mypy is only used for the cdk right now, but we can decide to use for connectors too if we decide it's useful